zio-http/src/it/scala/zhttp/AllApis.scala

```diff
@@ -0,0 +1,28 @@
+package zhttp
+
+import zhttp.http._
+import zhttp.socket.{Socket, WebSocketFrame}
+import zio.ZIO
+import zio.clock.Clock
+import zio.duration.durationInt
+import zio.stream.ZStream
+
+object AllApis {
+  def socket = Socket.collect[WebSocketFrame] { case WebSocketFrame.Ping =>
+    ZStream.succeed(WebSocketFrame.pong)
+  }
+
+  def api = HttpApp.collect[Any, Nothing] {
+    case Method.GET -> !!                   => Response.ok
+    case Method.POST -> !!                  => Response.status(Status.CREATED)
+    case Method.GET -> !! / "continue"      => Response.ok
+    case Method.GET -> !! / "boom"          => Response.status(Status.INTERNAL_SERVER_ERROR)
+    case Method.GET -> !! / "subscriptions" => Response.socket(socket)
+  }
+
+  def apiM = HttpApp.collectM[Clock, Nothing] { case Method.GET -> !! / "timeout" =>
```

Comment:
this is not needed as we will add a test for timeout middleware once that is merged